google/iam/admin/v1/iam_gapic.yaml

```diff
@@ -0,0 +1,248 @@
+type: com.google.api.codegen.ConfigProto
+language_settings:
+  java:
+    package_name: com.google.cloud.iam.admin.spi.v1
+  python:
+    package_name: google.cloud.iam.admin.v1
+  go:
+    package_name: cloud.google.com/go/iam/admin/apiv1
+  csharp:
+    package_name: Google.Iam.Admin.V1
+  ruby:
+    package_name: Google::Cloud::Iam::Admin::V1
+  php:
+    package_name: Google\Cloud\Iam\Admin\V1
+  nodejs:
+    package_name: '@google-cloud/admin'
```

Comment:
This will -- or should -- be the package name used by NPM too. So it's better to be consistent with other languages. If it's `google-cloud-iam` package in other languages, then it will be `@google-cloud/iam`. If they are `google-cloud-iam-admin`, it's better to be `@google-cloud/iam-admin`.

Note that NPM package names are not hierarchical at all. Normally package name can't contain `/` at all, but a format of `@(org name)/(package name)` is allowed (and `org name` still can't contain `/`).